.github/workflows/.bazelrc

```diff
@@ -0,0 +1,10 @@
+common --curses=no
+
+build --verbose_failures
+build --worker_verbose
+
+build --repository_cache=.cache
+fetch --repository_cache=.cache
```

Comment:
`--repository_cache` is a `build` option, it won't affect `fetch` as `fetch` doesn't inherit the `build` options https://docs.bazel.build/versions/master/command-line-reference.html#flag--repository_cache

I think

Reply:
Actually it is in [the Fetch Options section](https://docs.bazel.build/versions/master/command-line-reference.html#fetch) and in [the Query Options one](https://docs.bazel.build/versions/master/command-line-reference.html#query-options).

Reply:
Oh, didn't realize they were options that are literally declared in multiple sections 😮